### PR TITLE
Add sl perf page

### PR DESCRIPTION
**Pull request by xlight05**
## Purpose
Adds a page to show swan lake performance against other languages. Also adds a brief explanation about the performance framework. 

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by anuruddhal — APPROVED** [review]

File: docs/index.html [hunk 1]
```diff
@@ -32,66 +32,30 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+
+            <h5>Test Execution</h5>
+            <ol>
+                <li>It creates a fresh single node cluster(Standard_F4s_v2) Kubernetes cluster and installs Nginx ingress.</li>
+                <li>Deploy the application pod into the cluster (Hello world service written by different languages with 1 cpu, 512mb memory resource limits).</li>
+                <li>Deploy the JMeter pod and execute the benchmark and then commit the results to Github.</li>
+                <li>Cleanup the cluster.</li>
+            </ol>
+            <h5>Performance Accuracy</h5>
+            <p>Due to infrastructure-related reasons, we have noticed about 10% variance in worst-case test execution. Additionally, we haven't optimized any application for optimal settings and only wrote a simple hello world as our objective is simply to get a rough idea of where we are. This is not an effort to get the most accurate numbers. You could view the source codes(and other artifacts) for scenarios <a href="https://github.com/ballerina-platform/ballerina-performance-cloud/tree/main/load-tests">here.</a></p>
```
Comment (xlight05) [inline docs/index.html:58]: @praneesha  can you please review this?

File: docs/index.html [hunk 2]
```diff
@@ -32,66 +32,30 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
-        </div>
-        <div class="col-sm-8" id="date-selector">
-            From: <input class="ui-widget" id="fromDate" type="datetime-local">
-            To: <input class="ui-widget" id="toDate" type="datetime-local">
+            <h5>Purpose of the framework</h5>
+            <p>This performance framework was developed to get an idea of how ballerina performance changes with nightly builds and how ballerina performs compared to other languages in different use cases. </p>
+
+            <h5>Test Execution</h5>
+            <ol>
+                <li>It creates a fresh single node cluster(Standard_F4s_v2) Kubernetes cluster and installs Nginx ingress.</li>
```
Comment (anuruddhal) [inline docs/index.html:52]: Shall we add `Azure` as the environment as well

File: docs/daily-tests.html [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+        d3.csv(url).then(function (stats) {
+            //Throughput Chart
+            const throughput_dataset = getCSVData(stats, 'Throughput', chartTypes)
+            throughputChart = drawChart(throughput_dataset, 'throughputCanvas');
+
+            // Average Chart
+            const average_dataset = getCSVData(stats, 'Average', chartTypes);
+            averageChart = drawChart(average_dataset, 'averageCanvas');
+
+            //Standard Deviation of Response Time Chart
```
Comment (praneesha) [inline docs/daily-tests.html:340]:
```suggestion
            //Standard Deviation of Response Time Chart
```
Better to remove title case in code comments.

File: docs/index.html [hunk 4]
```diff
@@ -32,66 +32,30 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
-                <input class="period btn-light" id="3m" type="button" value="3m"/>
-                <input class="period btn-light" id="6m" type="button" value="6m"/>
-                <input class="period btn-light" id="1y" type="button" value="1y"/>
-            </div>
-        </div>
-        <div class="col-sm-8" id="date-selector">
-            From: <input class="ui-widget" id="fromDate" type="datetime-local">
-            To: <input class="ui-widget" id="toDate" type="datetime-local">
+            <h5>Purpose of the framework</h5>
+            <p>This performance framework was developed to get an idea of how ballerina performance changes with nightly builds and how ballerina performs compared to other languages in different use cases. </p>
```
Comment (praneesha) [inline docs/index.html:48]:
```suggestion
            <p>This performance framework was developed to get an idea of how Ballerina performance changes with nightly builds and how Ballerina performs compared to other languages in different use cases. </p>
```

File: docs/index.html [hunk 5]
```diff
@@ -32,66 +32,30 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+
+            <h5>Test Execution</h5>
+            <ol>
+                <li>Create a fresh single node cluster(Standard_F4s_v2) Kubernetes cluster in Azure and installs Nginx ingress.</li>
```
Comment (praneesha) [inline docs/index.html:52]:
```suggestion
                <li>Create a fresh single node (Standard_F4s_v2) Kubernetes cluster in Azure and installs Nginx Ingress.</li>
```

File: docs/index.html [hunk 6]
```diff
@@ -32,66 +32,30 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+                <li>Deploy the application pod into the cluster (Hello world service written by different languages with 1 cpu, 512mb memory resource limits).</li>
```
Comment (praneesha) [inline docs/index.html:53]:
```suggestion
                <li>Deploy the application pod into the cluster (i.e. into the `Hello world` service written by different languages with 1 cpu, 512mb memory resource limits).</li>
```

File: docs/index.html [hunk 7]
```diff
@@ -32,66 +32,30 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+                <li>Deploy the JMeter pod and execute the benchmark and then commit the results to Github. (Concurrent Users - 50)</li>
```
Comment (praneesha) [inline docs/index.html:54]:
```suggestion
                <li>Deploy the JMeter pod and execute the benchmark and then, commit the results to GitHub. (Concurrent Users - 50)</li>
```

File: docs/index.html [hunk 8]
```diff
@@ -32,66 +32,30 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+                <li>Cleanup the cluster.</li>
+            </ol>
+            <h5>Performance Accuracy</h5>
+            <p>Due to infrastructure-related reasons, we have noticed about 10% variance in worst-case test execution. Additionally, we haven't optimized any application for optimal settings and only wrote a simple hello world as our objective is simply to get a rough idea of where we are. This is not an effort to get the most accurate numbers. You could view the source codes(and other artifacts) for scenarios <a href="https://github.com/ballerina-platform/ballerina-performance-cloud/tree/main/load-tests">here.</a></p>
```
Comment (praneesha) [inline docs/index.html:58]:
```suggestion
            <p>Due to infrastructure-related reasons, a 10% variance was noticed in worst-case test execution. Additionally, any application is not optimized for optimal settings and only a simple hello world service is written for you to get a rough idea. This is not an effort to get the most accurate numbers. You could view the source codes (and other artifacts) for scenarios <a href="https://github.com/ballerina-platform/ballerina-performance-cloud/tree/main/load-tests">here.</a></p>
```
Do not use the words `we` and `our. Instead, use `you`.

Comment (praneesha) [reply docs/index.html:58]: Also, do not use the word `here` to give links.

Comment (praneesha) [inline docs/index.html:58]:
```suggestion
            <p>Due to infrastructure-related reasons, we have noticed about 10% variance in worst-case test execution. Additionally, we haven't optimized any application for optimal settings and only wrote a simple hello world as our objective is simply to get a rough idea of where we are. This is not an effort to get the most accurate numbers. For the source code (and other artifacts) of the scenarios. see <a href="https://github.com/ballerina-platform/ballerina-performance-cloud/tree/main/load-tests">load tests.</a></p>
```

File: docs/index.html (hunk 6 above)
Comment (xlight05) [reply docs/index.html:53]: Changed this to a different one. please check

File: docs/daily-tests.html [hunk 9]
```diff
@@ -329,23 +329,23 @@ <h3>Error Percentage</h3>
...
             const throughput_dataset = getCSVData(stats, 'Throughput', chartTypes)
             throughputChart = drawChart(throughput_dataset, 'throughputCanvas');
 
-            // Average Chart
+            //Average chart.
             const average_dataset = getCSVData(stats, 'Average', chartTypes);
             averageChart = drawChart(average_dataset, 'averageCanvas');
 
-            //Standard Deviation of Response Time Chart
+            //Standard Deviation of Response Time chart.
```
Comment (praneesha) [inline docs/daily-tests.html:340]: //Standard deviation of response time chart.

File: docs/daily-tests.html [hunk 10]
```diff
@@ -329,23 +329,23 @@ <h3>Error Percentage</h3>
...
             const deviation_dataset = getCSVData(stats, 'Std. Dev.', chartTypes)
             stdDevChart = drawChart(deviation_dataset, 'stdDevCanvas')
 
-            //99 percentile of Response Time Chart
+            //99 percentile of Response Time chart.
```
Comment (praneesha) [inline docs/daily-tests.html:344]: The 99th percentile of the response time chart.

File: docs/daily-tests.html [hunk 11]
```diff
@@ -329,23 +329,23 @@ <h3>Error Percentage</h3>
...
             const percentile_dataset = getCSVData(stats, '99% Line', chartTypes)
             percentileChart = drawChart(percentile_dataset, "percentileCanvas");
 
-            //Error Count Chart
+            //Error Count chart.
```
Comment (praneesha) [inline docs/daily-tests.html:348]: Error count chart.

File: docs/index.html [hunk 12]
```diff
@@ -45,17 +45,26 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+            <p>This performance framework was developed to get an idea of how Ballerina performance changes with nightly builds and how Ballerina performs compared to other languages in different use cases. </p>
 
-            <h5>Test Execution</h5>
+            <h5>Test Execution for Hello World Sample</h5>
             <ol>
-                <li>Create a fresh single node cluster(Standard_F4s_v2) Kubernetes cluster in Azure and installs Nginx ingress.</li>
-                <li>Deploy the application pod into the cluster (Hello world service written by different languages with 1 cpu, 512mb memory resource limits).</li>
-                <li>Deploy the JMeter pod and execute the benchmark and then commit the results to Github. (Concurrent Users - 50)</li>
+                <li>Create a fresh single node (Standard_F4s_v2) Kubernetes cluster in Azure and installs Nginx Ingress.</li>
+                <li>For each language hello world sample</li>
```
Comment (praneesha) [inline docs/index.html:53]: This seems to be incomplete.

File: docs/index.html [hunk 13]
```diff
@@ -45,17 +45,26 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+                <ol>
+                    <li>Deploy the application pod into the cluster (Hello world sample with 1 cpu, 512mb memory resource limits).</li>
```
Comment (praneesha) [inline docs/index.html:55]: 1 CPU, 512 MB

File: docs/index.html [hunk 14]
```diff
@@ -45,17 +45,26 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+                    <li>Deploy the JMeter pod and execute the benchmark and then, commit the results to Github. (Concurrent Users - 50)</li>
```
Comment (praneesha) [inline docs/index.html:56]: Should it be `POD`?

File: docs/index.html [hunk 15]
```diff
@@ -32,66 +32,39 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+                    <li>Clean up the application pod</li>
+                </ol>
+                <li>Cleanup the cluster.</li>
+            </ol>
+            <h5>Performance Accuracy</h5>
+            <p>A variance 10% was observed due to infrastructure-related reasons in worst-case between two test
```
Comment (praneesha) [inline docs/index.html:62]:
```suggestion
            <p>A variance of 10% was observed due to infrastructure-related reasons in the worst-case between two test 
```

File: docs/index.html [hunk 16]
```diff
@@ -32,66 +32,39 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
...
+                    <li>Deploy the application pod into the cluster (Hello world sample with 1 cpu, 512mb memory resource limits).</li>
+                    <li>Deploy the JMeter pod and execute the benchmark and then, commit the results to Github. (Concurrent Users - 50)</li>
+                    <li>Clean up the application pod</li>
+                </ol>
+                <li>Cleanup the cluster.</li>
+            </ol>
+            <h5>Performance Accuracy</h5>
+            <p>A variance 10% was observed due to infrastructure-related reasons in worst-case between two test 
+                executions. The samples are very simple hello world applications and the language specific optimizations
+                has not applied to any sample or deployments. This is only a high level use case specific test
```
Comment (praneesha) [inline docs/index.html:64]:
```suggestion
                have not applied to any sample or deployment. This is only a high level use case specific test 
```

File: docs/index.html (hunk 12 above)
Comment (xlight05) [reply docs/index.html:53]: This specifies a loop.  Modified to make it bit more readable. Please see if its okay

File: docs/index.html (hunk 14 above)
Comment (xlight05) [reply docs/index.html:56]: yes

**Review by praneesha — APPROVED** [review]

**Pull request merged**